tests/bin/check_tron_jobs_test.py

```diff
@@ -1554,22 +1559,44 @@ def test_sort_runs_by_interval_day(self):
 
     @patch('time.time', mock.Mock(return_value=1539633600.0), autospec=None)
     def test_sort_runs_by_interval_day_empty_buckets(self):
-        self.job['runs'].append({
-            'id': f"{self.job_name}.6",
+        self.job['runs'] = [{
+            'id': f"{self.job_name}.16",
             'job_name': self.job_name,
-            'run_num': 5,
+            'run_num': 16,
             'run_time': '2018-10-15 12:00:00',
             'start_time': '2018-10-15 12:00:00',
             'end_time': '2018-10-15 12:30:00',
             'state': 'succeeded',
             'exit_status': 0,
-        })
+        }] + self.job['runs']
 
         run_buckets = check_tron_jobs.sort_runs_by_interval(self.job, 'day')
 
         assert '2018.10.14' in run_buckets
         assert run_buckets['2018.10.14'] == []
 
+    @patch('time.time', mock.Mock(return_value=1539633600.0), autospec=None)
+    def test_sort_runs_by_interval_day_old_empty_buckets(self):
+        # If the newest run is a backfill for an older date, shouldn't be
+        # included for buckets
+        self.job['runs'] = [{
```

Comment:
This doesn't pollute other tests because fixtures are recreated... right?

Reply:
Yeah, the default scope for fixtures is per-function.